config/core/test-configs.yaml

```diff
@@ -868,6 +868,14 @@ device_types:
     class: arm64-dtb
     boot_method: uboot
 
+  fvp:
+    mach: vexpress
+    arch: arm64
+    dtb: 'fvp-base-revc.dtb'
```

Comment:
Should be:
   dtb: 'arm/fvp-base-revc.dtb'

Reply:
Thanks for the feedback, I have made the changes.